static/js/iframe-common.js

```diff
@@ -1,15 +1,23 @@
 require('iframe-resizer');
 
-export function generateIFrame(domain, queryParam, urlParam, token) {
+/**
+ * @typedef {import('./hitchhikerconfigmanager')} HitchhikerConfigManager
+ */
+
+/**
+ * Puts on iframe on the page of an Answers experience and sets up resizing and cross-domain communication
+ * 
+ * @param {string} domain The location of the answers experience
+ * @param {HitchhikerConfigManager} hitchhikerConfigManager used for passing dyamic config to the iframe
+ */
+export function generateIFrame(domain, hitchhikerConfigManager) {
   var isLocalHost = window.location.host.split(':')[0] === 'localhost';
   var containerEl = document.querySelector('#answers-container');
   var iframe = document.createElement('iframe');
   var pathToIndex = containerEl.dataset.path;
   iframe.allow = 'geolocation';
 
   domain = domain || '';
-  queryParam = queryParam || 'query';
```

Comment:
I removed these because I could not find anywhere that they were being used

Reply:
untangle one 🍝  a day to keep the smell away